src/gp-model/main.py

```diff
@@ -105,8 +110,13 @@ def train_model(X, Y, airqloud):
     Xset = X[keep,:]
     print('Number of rows in Xset', Xset.shape[0])
     
-    Xtraining = Xset[::2,:]
-    Ytraining = Yset[::2,:]
+    if Xset.shape[0]>9000:
+        Xtraining = Xset[::2,:]
+        Ytraining = Yset[::2,:]
+    else:
+        Xtraining = Xset
+        Ytraining = Yset
+    print('Number of rows in Xtraining', Xtraining.shape[0])
     
     if airqloud == 'kampala':
```

Comment:
Again, trying to appreciate the flow of things here.....
- Also, out of curiosity, what could be the motivation for singling out Kampala and Kira in this logic?
- And how shall new AirQlouds fit into this logic? Will they be in the else?

Reply:
The airqlouds that we have already have particular models that have been optimized to specifically fit them. Any airqloud for which this hasn't been done will fall in the else.

Reply:
Okay, make sense.